src/sentry/static/sentry/app/components/links/listLink.tsx

```diff
@@ -10,16 +10,15 @@ type DefaultProps = {
   activeClassName: string;
 };
 
-type Props = DefaultProps & {
-  to: LocationDescriptor;
-  className?: string;
-  query?: string;
-  onClick?: () => void;
-  // If supplied by parent component, decides whether link element
-  // is "active" or not ... overriding default behavior of strict
-  // route matching
-  isActive?: (location: LocationDescriptor, indexOnly?: boolean) => boolean;
-};
+type Props = DefaultProps &
+  React.ComponentProps<typeof Link> & {
+    to: LocationDescriptor;
```

Comment:
The link already has the `to` prop, being the prop of type `LocationDescriptor | ToLocationFunction`. I believe that we should either omit this prop or remove this line